slick-codegen/src/main/scala/slick/codegen/OutputHelpers.scala

```diff
@@ -29,18 +29,20 @@ trait OutputHelpers{
    *  Ensures the file ends with a newline character.
    *  @group Output
    */
-  def writeStringToFile(content: String, folder:String, pkg: String, fileName: String): Unit = {
-    val folder2 : String = folder + "/" + (pkg.replace(".", "/")) + "/"
+  def writeStringToFile(content: String, folder: String, pkg: String, fileName: String): Unit = {
+    val folder2: String = folder + "/" + pkg.replace(".", "/") + "/"
```

Comment:
I didn't touch this part really. Is that a request to change? Does the PR look good otherwise and ready to be merged if this nearby code gets improved?
Wondering..